src/python/WMCore/Services/PhEDEx/PhEDEx_DBS.py

```diff
@@ -0,0 +1,43 @@
+"""
+Module which depends on both PhEDEx and DBS.
+It provides function to create XML struction for injection.
+"""
+
+from __future__ import division
+
+from WMCore.Services.PhEDEx.XMLDrop import XMLInjectionSpec
+from WMCore.Services.DBS.DBSReader import DBSReader
```

Comment:
This is legacy code to support DBS2, since refactoring it, we can use
DBS3Reader